xbmc/cores/DllLoader/exports/emu_msvcrt.h

```diff
@@ -111,7 +111,8 @@ extern "C"
   int dll_feof (FILE * stream);
   int dll_fread (void * buffer, size_t size, size_t count, FILE * stream);
   int dll_getc (FILE * stream);
```

Comment:
Add to it on the top of header
#ifndef errno_t
#define errno_t int
#endif
to fix compile error on linux